README.md

```diff
@@ -1,2 +1,183 @@
-# terraform-azure-backup
-Highly opinionated management of Kubernetes backup with velero on AKS (Azure Kubernetes Service)
+# Terraform management of velero backup on AKS
+
+## Introduction
+
+This module manages velero backup on AKS (Azure Kubernetes Service)
+
+## Usage
+
+Instantiate the module by calling it from Terraform like this:
+
+```hcl
+module "azure-basics" {
+  source  = "dodevops/velero/azure"
+  version = "<version>"
+}
+```
+
+<!-- BEGIN_TF_DOCS -->
+## Requirements
+
+No requirements.
+
+## Providers
+
+The following providers are used by this module:
+
+- azurerm
+
+- helm
+
+- kubernetes
+
+## Modules
+
+No modules.
+
+## Resources
+
+The following resources are used by this module:
+
+- [azurerm_storage_account.storaccbackup](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_account) (resource)
+- [azurerm_storage_container.storcontbackup](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_container) (resource)
+- [helm_release.velero](https://registry.terraform.io/providers/hashicorp/helm/latest/docs/resources/release) (resource)
+- [kubernetes_namespace.velero](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/namespace) (resource)
+
+## Required Inputs
+
+The following input variables are required:
```

Comment:
This line does not add useful information to the descriptive title. This is a template comment, since it's valid for more than one description in this file.

Reply:
That's what [terraform-docs](https://github.com/terraform-docs/terraform-docs) does and we keep it for simplicity's sake.